composer.json

```diff
@@ -44,13 +44,15 @@
         ],
         "psr-4": {
             "App\\": "app/",
+            "App\\Plugins\\": "app/Plugins/",
             "Database\\Factories\\": "database/factories/",
             "Database\\Seeders\\": "database/seeders/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Tests\\": "tests/"
+            "Tests\\": "tests/",
+            "App\\Plugins\\": "tests/assets/Plugins/"
```

Comment:
In tests we use this directory. Which was not detected by the auto loader when adding the scopes to the test plugins.
With this the tests should run without problems.